index.html

```diff
@@ -64,6 +65,7 @@
       <a href="/faq">FAQ</a>
     </nav>
   </header>
+
```

Comment:
Purpose of the extra whitespace? It's already readable (and you can collapse the tags anyways)